minidump/src/minidump.rs

```diff
@@ -4726,11 +4726,20 @@ impl fmt::Display for CrashReason {
                     if let Some(mach_port_flavor) =
                         err::ExceptionCodeMacGuardMachPortFlavor::from_u64(flavor)
                     {
+                        // TODO: GUARD_EXC_STRICT_REPLY, GUARD_EXC_MOD_REFS and GUARD_EXC_IMMOVABLE have additional flags defined here:
```

Comment:
I forget if we ever properly enforced this style but imo ideally "TODO" is "blocks merging" and "FIXME" is "one of these days..."

Reply:
Gotcha, I'm changing it